internal/api/oidc/key.go

```diff
@@ -22,14 +25,33 @@ import (
 	"github.com/zitadel/zitadel/internal/zerrors"
 )
 
+var supportedWebKeyAlgs = []string{
+	// string(jose.EdDSA), Currently not supported in our OIDC library.
```

Comment:
we should disable it in the proto (key creation) as well for the moment to prevent issues

Reply:
If we can merge https://github.com/zitadel/oidc/pull/638 it EdDSA / ed25519 will work fine. I will remove this comment.